pandas/tests/scalar/timestamp/test_formats.py

```diff
@@ -0,0 +1,9 @@
+from pandas import Timestamp
+
+
+def test_isoformat():
```

Comment:
can you locate with with other formatting tests

Reply:
Can you suggest a more specific location that would be better?  I looked, but couldn't find an obvious place where other formatting tests are located.